[2021-08-16 15:47:27,352] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.create_tag_template_result 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:27,703] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.create_tag_template_result 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:27,755] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:27,769] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-08-16 15:47:27,773] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:27,974] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): create_tag_template_result> on 2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:28,034] {standard_task_runner.py:52} INFO - Started process 406 to run task
[2021-08-16 15:47:28,175] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_complex', 'create_tag_template_result', '2021-08-16T15:43:47.414310+00:00', '--job-id', '39', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpmxvfkx0h', '--error-file', '/tmp/tmpz0ajzweu']
[2021-08-16 15:47:28,253] {standard_task_runner.py:77} INFO - Job 39: Subtask create_tag_template_result
[2021-08-16 15:47:31,783] {logging_mixin.py:104} INFO - Running <TaskInstance: example_complex.create_tag_template_result 2021-08-16T15:43:47.414310+00:00 [running]> on host 8f195762143d
[2021-08-16 15:47:32,314] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=create_tag_template_result
AIRFLOW_CTX_EXECUTION_DATE=2021-08-16T15:43:47.414310+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:32,349] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-08-16 15:47:32,358] {bash.py:158} INFO - Running command: echo create_tag_template_result
[2021-08-16 15:47:32,407] {bash.py:169} INFO - Output:
[2021-08-16 15:47:32,408] {bash.py:173} INFO - create_tag_template_result
[2021-08-16 15:47:32,410] {bash.py:177} INFO - Command exited with return code 0
[2021-08-16 15:47:32,481] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_tag_template_result, execution_date=20210816T154347, start_date=20210816T154727, end_date=20210816T154732
[2021-08-16 15:47:33,279] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-08-16 15:47:33,286] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 406
[2021-08-16 15:47:33,664] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-08-16 15:47:33,668] {bash.py:185} INFO - Sending SIGTERM signal to bash process group
[2021-08-16 15:47:33,756] {process_utils.py:66} INFO - Process psutil.Process(pid=406, status='terminated', exitcode=1, started='15:47:27') (406) terminated with exit code 1
